cypress/global.d.ts

```diff
@@ -0,0 +1,12 @@
+declare namespace Cypress {
+  interface Chainable {
+    /**
+     * Custom command to select DOM element by data-test attribute.
+     * @example cy.getBySel('greeting')
+     */
+    getBySel(
+      dataTestAttribute: string,
+      args?: any
```

Comment:
Perhaps you've seen this one already, but an `any` is not allowed. Is there a type for `args`?

Reply:
This is from the official Cypress docs. I think I will remove the args, I'm not sure if I would use it.

Reply:
Kept it, just in case we need it in the future.